### PR TITLE
Record J2CL attachment parity closeout evidence

### PR DESCRIPTION
**Pull request by vega113**
## Summary
- update the J2CL/GWT parity matrix with the #971 deferred edge-case addendum
- remove block quote from the R-5.7 daily example set because it was not present in the inspected GWT `EditToolbar.java` action set
- add #971 local verification evidence and explicitly avoid claiming final browser-harness closure while #1025/#1026 remain open

## Verification
- `sbt -batch j2clSearchTest j2clProductionBuild j2clLitTest j2clLitBuild` - passed; Lit reported 23 files / 110 tests / 0 failures; SBT exited 0 while emitting the Vertispan DiskCache warning now tracked by #1027
- `python3 scripts/assemble-changelog.py && python3 scripts/validate-changelog.py` - passed; assembled 255 entries
- `git diff --cached --check` / `git diff --check` - passed
- `bash scripts/worktree-boot.sh --port 9971` - passed; staged GWT/J2CL assets
- `PORT=9971 ... bash scripts/wave-smoke.sh start` - reached `PROBE_HTTP=200` / `READY`, but `wave-smoke.sh check` failed after the staged process exited; #1026 tracks the persistent server lifecycle issue
- immediate route probe while the staged process was alive passed for `/`, `/?view=j2cl-root`, `/j2cl/index.html`, `/j2cl-search/sidecar/j2cl-sidecar.js`, and `/webclient/webclient.nocache.js`

## Review
- self-review: staged diff scope and whitespace checks passed
- Claude Opus 4.7 review loop: rounds 1-3 addressed clarity/nit concerns; round 4 verdict was ship-ready with zero required comments

Refs #971
Refs #904
Refs #1025
Refs #1026
Refs #1027

## Summary by CodeRabbit

* **Documentation**
  * Updated parity matrix metadata and clarified daily rich-edit example scope (removed block-quote from the daily examples) and added deferred edge-case entries for toolbar affordances, drag-and-drop attachments, client-side image compression, and extra mobile/touch gestures.
  * Added a local verification record summarizing test runs, build/check outcomes, startup readiness observations, diagnostic findings, and noted follow-up items to complete parity verification.

### CODE REVIEW

File: journal/local-verification/2026-04-25-issue-971-parity-closeout.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+- Worktree: /Users/vega/devroot/worktrees/codex-issue-971-parity-closeout
+- Date: 2026-04-25
+- Issue: #971
+
+## Commands
+
+- `bash scripts/worktree-boot.sh --port 9971`
+- `sbt -batch j2clSearchTest j2clProductionBuild j2clLitTest j2clLitBuild`
+- `python3 scripts/assemble-changelog.py && python3 scripts/validate-changelog.py`
+- `git diff --check`
```
[the remark this answers is not in the document]
Comment (vega113) [reply journal/local-verification/2026-04-25-issue-971-parity-closeout.md:15]: Fixed in c34bb455d.

File: journal/local-verification/2026-04-25-issue-971-parity-closeout.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
+# Local Verification
+
+- Branch: codex/issue-971-parity-closeout
+- Worktree: /Users/vega/devroot/worktrees/codex-issue-971-parity-closeout
+- Date: 2026-04-25
+- Issue: #971
```
[the remark this answers is not in the document]
Comment (vega113) [reply journal/local-verification/2026-04-25-issue-971-parity-closeout.md:6]: Fixed in c34bb455d.

File: docs/j2cl-gwt-parity-matrix.md [hunk 3]
```diff
@@ -221,6 +221,19 @@ recorded here so it is not silently dropped:
   example: rare legacy formatting paths, one-off keyboard shortcuts that do
   not appear in the `#969`/`#971` packet) — to be revisited only after the
   gate closes or via a dedicated addendum packet linked from this section
+- #971 daily attachment/rich-edit closeout defers the GWT toolbar affordances
+  that were not part of the inspected daily command packet: superscript,
+  subscript, font size, font family, font color, highlight color / `backColor`,
+  and block quote. The block quote affordance was not present in the inspected
+  GWT `EditToolbar.java` action set, so it is not counted as a #971 daily
+  parity requirement unless a later packet adds it explicitly. This finding is
+  the reason R-5.7 no longer lists block quote as a daily-path example.
```
[the remark this answers is not in the document]
Comment (vega113) [reply docs/j2cl-gwt-parity-matrix.md]: Fixed in c34bb455d.
